server/src/main/java/org/opensearch/cluster/routing/allocation/ConstraintTypes.java

```diff
@@ -28,6 +28,11 @@ public class ConstraintTypes {
      */
     public final static String CLUSTER_PRIMARY_SHARD_BALANCE_CONSTRAINT_ID = "cluster.primary.shard.balance.constraint";
 
+    /**
+     * Defines a cluster constraint which is breached when a node contains more than avg primary shards across all indices
+     */
+    public final static String CLUSTER_PRIMARY_SHARD_REBALANCE_CONSTRAINT_ID = "cluster.primary.shard.balance.constraint";
```

Comment:
can we follow the pattern as there for other similar settings like `cluster.routing.allocation.balance.prefer_primary`, `cluster.routing.allocation.balance.shard`, `cluster.routing.allocation.balance.index`?